contactsummary/js/src/contact-summary/services/contract.js

```diff
@@ -180,6 +180,40 @@ define([
                 });
         };
 
+        /**
+         * @ngdoc method
+         * @name getLengthOfServiceYmd
```

Comment:
naming the method as getLengthOfServiceYmd is a bit akward ( to me ) .. I think getLengthOfService is better .